pyupgrade.py

```diff
@@ -669,6 +669,20 @@ def _is_bytestring(s):
         return False
 
 
+def _is_ascii(s):
+    if sys.version_info >= (3, 7):  # pragma: no cover (py37+)
+        return s.isascii()
+    else:  # pragma: no cover (<py37)
+        try:
+            s.encode('ascii')
+        except UnicodeDecodeError:  # pragma: no cover (py2)
+            return False
+        except UnicodeEncodeError:  # pragma: no cover (py2+)
+            return False
+        else:
+            return True
```

Comment:
some of these branches aren't necessary -- the tokens you get will always be text -- did this get copy pasted from somewhere?

Reply:
I think I was playing with b’’ strings in the interpreter. Will revisit this